roles/redhat_satellite6_capsule_lb/tasks/main.yml

```diff
@@ -0,0 +1,92 @@
+---
+# tasks file for satellite_lb
+
+- name: Generate LB Certs 1
+  when: inventory_hostname in groups['satellite']
+  shell: capsule-certs-generate --foreman-proxy-fqdn "{{ capsule1 }}" --certs-tar /tmp/capsule1.tar --foreman-proxy-cname "{{ lb_server }}"
+
+- name: Generate LB Certs 2
+  when: inventory_hostname in groups['satellite']
+  shell: capsule-certs-generate --foreman-proxy-fqdn "{{ capsule2 }}" --certs-tar /tmp/capsule2.tar --foreman-proxy-cname "{{ lb_server }}"
+
+- name: pull the cert from the {{ sat_server }} to the local machine
+  when: inventory_hostname in groups['satellite']
+  fetch:
+    flat: yes
+    src: /tmp/capsule1.tar
+    dest: /tmp/
+
+- name: pull the cert from the {{ sat_server }} to the local machine
+  when: inventory_hostname in groups['satellite']
+  fetch:
+    flat: yes
+    src: /tmp/capsule2.tar
+    dest: /tmp/
+
+- name: push cert to {{ capsule1 }}
+  when: inventory_hostname == "{{ capsule1 }}"
```

Comment:
@Smithe2413 rather then hard coding this to capsule1 and capsule2 couldn't this all be done in a loop for the number of hosts in the `lb_capsules` group, or some variable that gets put on a capsule host to specify which lb group it is in (maybe that is getting to fancy though)